modules/role/role.tf

```diff
@@ -1,3 +1,8 @@
+locals {
+  should_create_s3_policy = var.blacklist_object_arn != null && var.blacklist_object_arn != "" ? true : false
```

Comment:
isn't the ternary operator redundant here?
`? true : false` ...

Reply:
you can set `1: 0` instead of `true/false` then you don't need the if in #31,41,51

Reply:
another thing is the check for `""` I think we should not check for empty and apply if it is empty without a policy - if user is uaing it - he should provide the arn, otherwise - it fails (and not passing without the blacklist)

Reply:
@eladbash dropped the `""` check